scripts/rag_evaluation/create_evaluation_dataset/chunks_flatten_strategy/create_chunks.py

```diff
@@ -0,0 +1,63 @@
+import json
+import numpy as np
+from typing import List
+import matplotlib.pyplot as plt
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+from settings import OPENAI_MODEL_EMBEDDING
+
+
+text_splitter = RecursiveCharacterTextSplitter(
+    separators=["\n\n", "\n", " ", ""],
+    chunk_size=500,
+    chunk_overlap=50,
+    length_function=len,
+)
+
+
+def get_total_tokens_from_string(string: str,
+                                 encoding=OPENAI_MODEL_EMBEDDING) -> int:
+    """Returns the number of tokens in a text string."""
+    return len(encoding.encode(string))
+
+
+def measure_tokens_lenghts(file_path, tokens_lengths):
```

Comment:
this function seems to be duplicated several times in different files. You should prefer to implement it once and reuse